terraform/concourse/elb.tf

```diff
@@ -1,5 +1,5 @@
-data "aws_acm_certificate" "concourse" {
-  domain   = "${var.concourse_hostname}.${var.system_dns_zone_name}"
+data "aws_acm_certificate" "system_acm_cert" {
```

Comment:
I think the `_acm_cert` in the name here is redundant. It's also inconsistent with the naming in paas-cf.

Reply:
I will take care of that

Reply:
fixed